test/unit/server_test.js

```diff
@@ -670,21 +670,78 @@ describe("server.js tests", function () {
           })
       });
 
-      it("trades() requests the correct endpoint", function (done) {
+    });
+
+    describe("TradesCallBuilder", function() {
+      it("trades() requests the correct endpoint (no filters)", function (done) {
+        let response = "trades_no_filters";
```

Comment:
I've seen your comment about mocked responses in other PR but I think we should make them similar to real ones. For example we should test if functions are correctly created from `_links`.

Reply:
No prob.